src/store/metadata/stub/content/editions/total-value-of-uk-exports-2022-08.json

```diff
@@ -0,0 +1,130 @@
+      "creator": "ofcom",
+      "contact_point": {
+        "name": "Ofcom Open Data",
+        "email": "mailto:open.data@ofcom.org.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/economy"
+      ],
+      "frequency": "monthly",
+      "keywords": [
+        "economic-growth",
+        "exports",
+        "goods-and-services",
+        "imports",
+        "trade-deficit",
+        "subnational",
+        "international-territoral-level"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-05-23T10:14:20",
+      "modified": "2023-08-25T11:09:00",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1M",
+        "P3M",
+        "P1Y"
+      ],
+      "temporal_coverage": {
+        "start": "2022-09-01T00:00:00Z",
+        "end": "2022-09-30T23:59:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions/1",
+          "issued": "2023-05-23T10:14:20"
+        }
+      ],
+      "next_release": "2023-09-24T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Period",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The geography code associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name assiciated with The geography code "
+          },
+          {
+            "name": "geography",
+            "datatype": "string",
+            "titles": "Geography Type",
+            "description": "The categorisation of the geography code"
+          },
+          {
+            "name": "value",
+            "datatype": "decimal",
+            "titles": "Geographic area with Total value of UK exports by at least one provider",
+            "description": "Observational Value"
+          },
+          {
+            "name": "measure",
+            "datatype": "string",
+            "titles": "Measure Type",
+            "description": "Measure Type of the Observational Value"
+          },
+          {
+            "name": "unit",
+            "datatype": "string",
+            "titles": "Unit",
+            "description": "Unit Type of the Observational Value"
+          },
+          {
+            "name": "lower confidence interval (95%)",
```

Comment:
These names should be consistent with the other recently updated datasets: lower_confidence_interval_95 instead of lower confidence interval (95%)

Reply:
okay